packages/foam-cli/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "foam-cli",
+  "name": "@foam/cli",
```

Comment:
Is the namespace/scope required for the TypeScript scoping? We don't own the `@foam` scope on npm so we can't publish the CLI by this name.

Reply:
good point I didn't think of that. will use `foam-core`, `foam-cli` and `foam-vscode`